### ISSUE TITLE
train.py bugs

### ISSUE THREAD

**Issue by z393** [maudzung/CenterNet3D-PyTorch#2]
>>> configs: {'seed': 2020, 'saved_fn': 'centernet3d', 'root_dir': '../', 'arch': 'centernet3d', 'pretrained_path': None, 'hflip_prob': 0.0, 'no_val': False, 'num_samples': None, 'num_workers': 4, 'batch_size': 16, 'print_freq': 50, 'tensorboard_freq': 50, 'checkpoint_freq': 2, 'start_epoch': 1, 'num_epochs': 50, 'lr_type': 'one_cycle', 'lr': 0.00225, 'minimum_lr': 1e-07, 'momentum': 0.949, 'weight_decay': 0.01, 'optimizer_type': 'adam', 'steps': [150, 180], 'world_size': -1, 'rank': -1, 'dist_url': 'tcp://127.0.0.1:29500', 'dist_backend': 'nccl', 'gpu_idx': 0, 'no_cuda': False, 'multiprocessing_distributed': False, 'evaluate': False, 'resume_path': None, 'K': 100, 'device': device(type='cuda', index=0), 'ngpus_per_node': 1, 'pin_memory': True, 'sparse_shape': [40, 1400, 1600], 'input_size': [1400, 1600], 'hm_size': [350, 400], 'down_ratio': 4, 'max_objects': 50, 'head_conv': 64, 'num_classes': 1, 'num_center_offset': 2, 'num_z': 1, 'num_dim': 3, 'num_direction': 2, 'num_conners': 4, 'heads': {'hm_cen': 1, 'cen_offset': 2, 'direction': 2, 'z_coor': 1, 'dim': 3, 'hm_conners': 1}, 'num_input_features': 4, 'dataset_dir': '../dataset/kitti', 'checkpoints_dir': '../checkpoints/centernet3d', 'logs_dir': '../logs/centernet3d', 'distributed': False, 'subdivisions': 4, 'is_master_node': True}
QObject::moveToThread: Current thread (0x5632b7c00070) is not the object's thread (0x5632b83c94d0).
Cannot move to target thread (0x5632b7c00070)

qt.qpa.plugin: Could not load the Qt platform plugin "xcb" in "/home/hy/anaconda3/envs/pointpillars/lib/python3.7/site-packages/cv2/qt/plugins" even though it was found.
This application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem.

Available platform plugins are: xcb, eglfs, linuxfb, minimal, minimalegl, offscreen, vnc, wayland-egl, wayland, wayland-xcomposite-egl, wayland-xcomposite-glx, webgl.

I met this error but I don't know how to deal it. Please help me
